libazureinit/src/provision/ssh.rs

```diff
@@ -277,7 +287,9 @@ mod tests {
             },
         ];
 
-        provision_ssh(&user, &keys, Some(".ssh/xauthorized_keys".to_string()))
+        let authorized_keys_path = user.dir.join(".ssh/xauthorized_keys");
```

Comment:
Is the file name supposed to be `xauthorized_keys` instead of `authorized_keys`?

Reply:
Ah, I see that this is in a test, never mind.